script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -73,76 +73,76 @@ The vampire's fighting spirit has not faded.
+"However, she was utterly undeserving of the position of family head.//だが、ご当主にとっても、ソレはより不相応だった。
+Therefore I took it. As a knight, I defeated the Queen and usurped her throne. And now this Principle is mine alone."//Referring to Zaria, so queen/her?
 "I see. It must have been a short reign."
 <Page31>
 "...Well, aren't you an arrogant one? You are so weak that you cannot even use a Marble Phantasm, and yet you decide to face me head-on?"
 "It doesn't matter. There is no need to materialize a world against a mere Dead Apostle.
 Against someone like you――― these claws are more than enough."
 "――――――"
 <Page32>
-The edges of the vampire's mouth slant upwards.
-His refined face turns into that of an ugly beast.
-He laughs at Arcueid's provocation in acknowledgment of her statement.
+The edges of the vampire's mouth twist upwards.
+His refined face warps into that of an ugly beast.//成り下がる
+As though he were grinning in agreement with Arcueid's provocation.//アルクェイドの挑発を、その通りだ、と<嗤|わら>うように。
 <Page33>
-"The strongest ones are those who continue to change and remain unbiased.
-...You are exactly that. Even as you decline to such an extent, you still have power remaining that surpasses my own.
-A primordial one that deals with every curse and every situation―――No ordinary Dead Apostle can possibly compare."
+"The strongest ones are those who continue to adapt and do not remain locked in their ways.//最も強きものとは千変にして偏|かたよ>らぬもの
+...You are exactly that. Even diminished as you are, your remaining power still surpasses my own.//そこまで衰退していながら、まだおれを上回る余力がある。
+A primordial one that deals with every curse and every situation――― No ordinary Dead Apostle can possibly compare."
 <Page34>
-The air becomes distorted.
-The blue fever of ill thoughts lowers the temperature underground.
+The atmosphere distorts.//空気が歪む
+The blue fever of ill thoughts lowers the temperature in the crater.//病想の青い熱が、地下の気温を下げていく。
 <Page35>
-Arcueid has no hesitation.
+Arcueid doesn't hesitate.
 She charges Vlov head-on, despite knowing that his strange weapon has the potential to defeat her.
 ―――And then.
 <Page36>
 "But you do not understand.
-Although her territory is a part of this planet, this is a biosphere that has been isolated as being unfit to live in.
+Although that territory is a part of this planet, it is an isolated biosphere in which life cannot exist.//I think this か is an inanimate reference here? <彼|か>の領土はこの星の一部でありながら、“生きるに<能|あた>わず”と除外された生命圏。
 Even a Princess of the True Ancestors like yourself cannot possibly endure a hell like this...!"
 <Page37>
-A cold wave of air spreads around Vlov.
-Arcueid comes to a sudden halt and sticks her hands out in order to protect herself.
-The wind blows fiercely, amplifying to a piercing scream, and envelopes the city in absolute zero.
+A wave of cold air spreads around Vlov.
+Arcueid comes to a sudden halt and sticks her hands out in order to protect herself.//Herself accurate here? 何かを守るように手を突き出すアルクェイド。
```

Comment:
The nuance could be that she wants to protect the city. 
Maybe 'Arcueid comes to a sudden halt and sticks her hands out in a protective gesture.'